strapi/strapi
# Update: Purchase Page links + Add Content History menu link

### What does it do?

- Replace Purchase page links.
- Add a Content History Menu Link for CE users only in order to display a purchase page.

### Why is it needed?

Give more visibility to our Enterprise features.

### How to test it?

- Start project in CE
You should see the Content History Menu link

- Start project in EE
You should not see the Content History Menu link.
